apps/hub-app/src/components/Avatar.tsx

```diff
@@ -0,0 +1,45 @@
+import * as RadixAvatar from '@radix-ui/react-avatar';
```

Comment:
I know that Jord is working on a composable Avatar for the component library. Once that's ready we can compare / update this component and his

Reply:
@jordanlesich we could reference this when building out #144 -- flagging it incase you don't see it